### PR TITLE
Added documentation to travis.yml

### LINKED ISSUE #864: travis.yml undocumented
**I'm submitting a ...**
- [ ] bug report
- [x] feature request

**Current behavior:**

The travis.yml file is completely undocumented, making it difficult to comprehend

**Expected behavior:**
- The file should have prover comments to explain the process

**Screenshots of the issue:**
https://github.com/fossasia/susper.com/blob/master/.travis.yml

### PR DESCRIPTION
**Pull request by YashKumarVerma**
Fixes #864 

#### Checklist

- [x] I have read the [Contribution & Best practices Guide](https://blog.fossasia.org/open-source-developer-guide-and-best-practices-at-fossasia) and my PR follows them.
- [x] My branch is up-to-date with the Upstream `master` branch.
- [x] I have added necessary documentation (if appropriate)

#### Changes proposed in this pull request:
Added inline documentation to travis

### CODE REVIEW

**Comment by YashKumarVerma** [comment]
@ParthS007 @fossasia/susper

File: .travis.yml [hunk 1]
```diff
@@ -1,29 +1,42 @@
+# Using a sudo enabled, Linux trusty distribution, VM
 sudo: required
 dist: trusty
+
+
```
Comment (harshit98) [inline .travis.yml]: Remove extra spaces. Provide only 1 space.

Comment (YashKumarVerma) [reply .travis.yml]: Sure

**Comment by YashKumarVerma** [comment]
@harshit98 @bhaveshAn @ParthS007 
Done the changes

**Comment by YashKumarVerma** [comment]
so can we have a merge ?

**Comment by YashKumarVerma** [comment]
@fossasia/susper

**Pull request merged**